influx/authorization.go

```diff
@@ -0,0 +1,93 @@
+package influx
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/influxdata/chronograf"
+)
+
+// Authorization adds optional authorization header to request
+type Authorization interface {
```

Comment:
Mind changing this to an agent noun of some variety to better align with convention (e.g. `io.Writer`, `http.Flusher`, etc.)? Maybe `Authorizer` ?